lib/controllers/orders.js

```diff
@@ -4,24 +4,24 @@ const pool = require('../utils/pool');
 
 module.exports = Router()
   .post('/', async (req, res) => {
-    const { rows } = await pool.query(
-      'INSERT INTO orders(product, quantity) VALUES ($1, $2) RETURNING *;',
-      [req.body.product, req.body.quantity]
-    );
-    const order = new Order(rows[0]);
-
-    res.json(order);
+    try {
+      const order = await Order.insert({
+        product: req.body.product,
+        quantity: req.body.quantity,
+      });
+      res.json(order);
+    } catch {
+      return null;
```

Comment:
Convention here would be to return null `if(!rows[0])` inside of the instance method of your Class instead